src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -354,7 +359,8 @@ export const CloudEnvironmentModal = ({
         div({ style: toolLabelStyles }, [
           img({
             src: getToolIcon(toolLabel),
-            style: { height: 20 }
+            style: { height: 20 },
+            alt: `${toolLabel} image`
```

Comment:
Just put the name of the tool, as now it is reading "Jupyter image, image" (the screen reader adds on image).

It is good to add the alt text here, but it's not actually the images I was talking about. The place that needs to be updated for the buttons on the context bar is `getImgForTool`, and line 133 for the general cloud icon.

![image](https://user-images.githubusercontent.com/484484/160690281-110d0b26-730e-4cad-b14e-0ad4330fc627.png)

Reply:
@jdcanas what is working the best here is to put an empty alt text attribute on these images. Then the screenreader will use the tooltip text as the name of the label.

If you put the tool's label as the alt text, you don't get to hear to tooltip text and lose the status information.